docs/reference/setup/install/check-running.asciidoc

```diff
@@ -5,7 +5,7 @@ You can test that your {es} node is running by sending an HTTPS request to port
 
 ["source","sh",subs="attributes"]
 ----
-curl --cacert {es-conf}{slash}config{slash}certs{slash}http_ca.crt -u elastic https://localhost:9200 <1>
+curl --cacert {es-conf}{slash}certs{slash}http_ca.crt -u elastic https://localhost:9200 <1>
```

Comment:
@shchedrin, thank you for fixing this issue! While you're correct that there isn't a `/confg` directory on DEB or RPM packages, there _is_ a `/config` directory in MacOS and Windows packages. We're using replacement variables in each of the installation files. For example in the [targz.asciidoc](https://github.com/elastic/elasticsearch/blame/8.1/docs/reference/setup/install/targz.asciidoc#L83) file for MacOS, we're replacing the `{es-conf}` and `{slash}` variables. We're doing the same thing in [rpm.asciidoc](https://github.com/elastic/elasticsearch/blame/8.1/docs/reference/setup/install/rpm.asciidoc#L145). If we remove `/config` from `check-running.asciidoc`, then it will be removed for all installation docs. This is complicated, but ensures that we can replace different parts of the path for each installation type. 

cc: @mark-vieira